NAMESPACE

```diff
@@ -1,4 +1,4 @@
 # Generated by roxygen2: do not edit by hand
 
+export(initial_sync)
```

Comment:
why rename this? I used `sync` because that's the name of the API call on the Matrix side, so it's consistent.

Reply:
I renamed it because of the clash between the variable name and the function name. But you are right, it's better to rename the variable instead (`initial_sync <- sync()`).